### PR TITLE
Trigger jobs based on stage and production addons new release

### PR DESCRIPTION
**Pull request by chetna14manku**
Changes to trigger jobs based on new merge of addons in stage and production. 
Update configmap like this:
```
products_jobs_mapping:
        rhods:
            stage:
              - periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-rosa-rhods-addon-baseline-fips
              - periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-stage-smoke-osd-rhods-addon-baseline-fips
            production:
              - periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-production-smoke-rosa-rhods-addon-baseline-fips
              - periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-GA-production-smoke-osd-rhods-addon-baseline-fips            
```

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -345,8 +349,10 @@ def _trigger_jobs(
                 changed_file,
             )
             if matches:
+                ocm_env = "stage" if "stage" in changed_file else "production"
```
Comment (myakove) [inline app/app.py:352]: `ocm_env` can be matches in the regex (line 347)

**Review by myakove — CHANGES_REQUESTED** [review]

File: app/app.py (hunk 1 above)
Comment (myakove) [reply app/app.py:352]: matches = re.match(
                 r"addons/(?P<product>.*)/addonimagesets/(?P<env>production|stage)/.*.yaml",
                 changed_file,
             )
matches.group("env")

Comment (chetna14manku) [reply app/app.py:352]: yes, thank you, updating the code!

**Review by rnetser — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**